lib/cdo/hamburger.rb

```diff
@@ -252,7 +252,7 @@ def self.get_header_contents(options)
       # When signed out, "Learn" will take all users to the student view of /courses.
       {title: I18n.t("#{loc_prefix}learn"), url: CDO.studio_url("/courses"), id: "header-learn"},
       {title: I18n.t("#{loc_prefix}teach"), url: CDO.code_org_url("/teach"), id: "header-teach"},
-      {title: I18n.t("#{loc_prefix}project_gallery"), url: CDO.studio_url("/projects/public"), id: "header-projects"},
+      {title: I18n.t("#{loc_prefix}schools"), url: CDO.code_org_url("/administrators"), id: "header-schools"},
       {title: I18n.t("#{loc_prefix}stats"), url: CDO.code_org_url("/promote"), id: "header-stats"},
       {title: I18n.t("#{loc_prefix}help_us"), url: CDO.code_org_url("/help"), id: "header-help"},
       {title: I18n.t("#{loc_prefix}incubator"), url: CDO.studio_url("/incubator"), id: "header-incubator"},
```

Comment:
Changes look good, just a cleanup comment.
Could we have the same translation patterns for all links? It can be confusing as to what pattern to use if someone isn't familiar with this code.

Sometimes we do:
```
{title: I18n.t("#{loc_prefix}i18n_key"), ...},
```
and sometimes:
```

[...].each do |entry|
      entry[:title] = I18n.t("#{loc_prefix}#{entry[:title]}")
end
```